src/biokbase/narrative/jobs/job.py

```diff
@@ -347,47 +355,63 @@ def state(self, force_refresh=False):
             state = self.query_ee2_state(self.job_id, init=False)
             self._update_state(state)
 
-        return self._internal_state(JOB_INIT_EXCLUDED_JOB_STATE_FIELDS)
+        return self._internal_state(exclude)
 
     def _internal_state(self, exclude=None):
         """Wrapper for self._acc_state"""
         state = copy.deepcopy(self._acc_state)
         self._trim_ee2_state(state, exclude)
         return state
 
-    def output_state(self, state=None) -> dict:
+    def output_state(self, state=None, no_refresh=False) -> dict:
```

Comment:
Add `no_refresh` parameter so that `jm._construct_job_state_set` can ask for the stagnant output state when EE2 won't cooperate